src/@next/Modal/ModalStyle.ts

```diff
@@ -76,7 +77,7 @@ export const StyledModalActions = styled.div`
   padding: ${space16} 20px;
   gap: 10px;
   box-shadow: inset 0px 1px 0px ${Neutral.B95};
-  border-radius: 0 ${borderRadius8};
+  border-radius: 0 0 ${borderRadius8} ${borderRadius8};
```

Comment:
fixing these ![Screenshot 2023-04-03 at 12 16 05 PM](https://user-images.githubusercontent.com/7514754/229410249-5b5afcf3-ef16-4438-9c56-4a2d96a5f5ff.png)

Reply:
what will the design look like if the action section below is not applied?

Reply:
@davidzzz this is an example https://aries.glints.design/?path=/story/next-modal--with-large-content